.NET/Microsoft.Recognizer.Text.TimeZone/Extractors/IExtractor.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Recognizers.Text.TimeZone
+{
+    public interface IExtractor
+    {
+        List<ExtractResult> Extract(string input);
+    }
+    public class ExtractResult
```

Comment:
Same applies to Result. Unless this is only some internal structure that will later be converted to the one returned to users.